website/static/js/mobile_nav.js

```diff
@@ -0,0 +1,67 @@
+(() => {
+  let isMainNavClick = false;
+  let isSubNavClick = false;
+  document.addEventListener('DOMContentLoaded', function() {
+    addClickHandlersToMainNavTriggers();
+    addClickHandlersToSubNavTriggers();
+  });
+
+  function addClickHandlersToMainNavTriggers() {
+    const mainNavToggles = document.querySelectorAll('.main-nav .mobile.main-nav-trigger');
+    const mainNav = document.querySelector('.main-nav .navigationWrapper.navigationSlider');
+    window.addEventListener('click', (event) => {
```

Comment:
I don't think arrow functions work with IE. Not positive though. I needed to remove all arrow functions and string templates from `segment.js`. We can wait for Beth to verify.

Reply:
Don't we have babel? Babel is supposed to handle compatibility issues with ES6

Reply:
This is a static file. It is served as is. So babel does not compile it.